lib/browser.js

```diff
@@ -19,122 +19,127 @@ var EXECUTING_DISCONNECTED = 4
 // The browser got permanently disconnected (being removed from the collection and destroyed).
 var DISCONNECTED = 5
 
-var Browser = function (id, fullName, /* capturedBrowsers */ collection, emitter, socket, timer,
-  /* config.browserDisconnectTimeout */ disconnectDelay,
-  /* config.browserNoActivityTimeout */ noActivityTimeout) {
-  var name = helper.browserFullNameToShort(fullName)
-  var log = logger.create(name)
-  var activeSockets = [socket]
-  var activeSocketsIds = function () {
-    return activeSockets.map(function (s) {
+class Browser {
+  constructor (id, fullName, /* capturedBrowsers */ collection, emitter, socket, timer,
+    /* config.browserDisconnectTimeout */ disconnectDelay,
+    /* config.browserNoActivityTimeout */ noActivityTimeout) {
+    this.name = helper.browserFullNameToShort(fullName)
+    this.log = logger.create(this.name)
+    this.activeSockets = [socket]
+    this.id = id
+    this.fullName = fullName
+    this.state = READY
+    this.lastResult = new Result()
+    this.disconnectsCount = 0
+    this.pendingDisconnect = null
+    this.timer = timer
+    this.noActivityTimeoutId = null
+    this.noActivityTimeout = noActivityTimeout
+    this.emitter = emitter
+    this.collection = collection
+    this.disconnectDelay = disconnectDelay
+    this.socket = socket
+  }
+  activeSocketsIds () {
+    return this.activeSockets.map(function (s) {
```

Comment:
You can use arrow function, like: `this.activeSockets.map(s => s.id).join(', ')`